setup.py

```diff
@@ -9,13 +9,19 @@
 requirements = ["pyyaml",
                 "nose>=1.3.7",
                 "future-fstrings",
-                "scabha @ git+https://github.com/caracal-pipeline/scabha2",
+                "scabha >= 0.7.0", 
```

Comment:
Then we should just remove the scabha dependency for now. Because scabha 0.7.0 is not on pypi, and it makes no sense to require it.

Reply:
Or we could just merge the scabha PR and make a release?

Reply:
That works for me